packages/art/package.py

```diff
@@ -111,11 +111,9 @@ def setup_dependent_build_environment(self, env, dependent_spec):
         # Cleaup.
         sanitize_environments(env, "CET_PLUGIN_PATH", "PERL5LIB")
 
-    def setup_dependent_run_environment(self, env, dependent_spec):
+    def setup_run_environment(self, env):
```

Comment:
This is already defined in line 96 above.

Reply:
Yes, I see that now;  the first several I went through only had a setup_dependent_run_environment, and no setup_run_environment, and I just assumed they were all that way..   I've tried to unify them properly.